sdk/wallet-adapter/packages/adapters/wallet-standard-adapter/src/index.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { WalletAdapterProvider } from "@mysten/wallet-adapter-base";
+import {
+  Wallets,
+  isStandardWalletAdapterCompatibleWallet,
+  StandardWalletAdapterWallet,
+} from "@mysten/wallet-standard";
+import { initialize } from "@wallet-standard/app";
+import { StandardWalletAdapter } from "./StandardWalletAdapter";
+import mitt, { Emitter } from "mitt";
+
+type Events = {
+  changed: void;
+};
```

Comment:
I assume this is a placeholder value?

Reply:
Nope, it's the type of the events that are emitted. `changed` is the name of the event, `void` is the data associated with the event (as in, there is no data for this event).